base_geoengine/geo_ir/ir_fields_converter.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Onestein
+from odoo import api, models
+
+
+class IrFieldsConverter(models.AbstractModel):
+    _inherit = 'ir.fields.converter'
+
+    @api.model
+    def _str_to_geo_polygon(self, model, field, value):
+        return value, []
+
+    _str_to_geo_point = _str_to_geo_polygon
+    _str_to_geo_line = _str_to_geo_polygon
+    _str_to_geo_multi_line = _str_to_geo_polygon
+    _str_to_geo_multi_point = _str_to_geo_polygon
+    _str_to_geo_multi_polygon = _str_to_geo_polygon
```

Comment:
@tarteo this change is part of the commit "[IMP] Fix unclickable overlapping features, importing geo data, opacity in map widget"

Could you explain to me what it does? Should it go in a separate commit?
I guess it's for importing data. But, by which mean? csv?

Reply:
Yes, this tells Odoo how to convert the string value in the CSV to something the ORM field understands.